src/RunQueryButtons.test.tsx

```diff
@@ -8,42 +8,42 @@ const getDefaultProps = (overrides?: Partial<RunQueryButtonsProps<DataQuery>>) =
   return {
     onRunQuery: jest.fn(),
     onCancelQuery: jest.fn(),
-    isQueryValid: jest.fn(),
+    enableRun: true,
     query: { refId: 'refId' },
     ...overrides,
   };
 };
 
 describe('RunQueryButtons', () => {
   it('disable the run button if the query is invalid', () => {
-    const props = getDefaultProps({ isQueryValid: jest.fn().mockReturnValue(false) });
+    const props = getDefaultProps({ enableRun: false});
     render(<RunQueryButtons {...props} />);
-    const runButton = screen.getByRole('button', { name: 'Run' });
+    const runButton = screen.getByRole('button', { name: 'Run query' });
     expect(runButton).toBeDisabled();
   });
 
   it('run button should be enabled if the query is valid', () => {
-    const props = getDefaultProps({ isQueryValid: jest.fn().mockReturnValue(true) });
+    const props = getDefaultProps({ enableRun: true});
```

Comment:
the test case description should be something like

'run button should be enabled if `enableRun` is true'